cmd/daytona/config/const.go

```diff
@@ -78,6 +79,8 @@ func GetDocsLinkFromGitProvider(providerId string) string {
 		return "https://docs.gitness.com/administration/user-management#generate-user-token"
 	case "azure-devops":
 		return "https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat"
+	case "aws-codecommit":
+		return "https://aws.amazon.com/codecommit/resources and to configure AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY & AWS_DEFAULT_REGION read https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html  "
```

Comment:
Instead of the resources URL, please put this https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up-gc.html